src/twister2/plugin.py

```diff
@@ -139,6 +143,18 @@ def pytest_addoption(parser: pytest.Parser):
              'To verify their current status, one can run only quarantined tests '
              'using mark: -m quarantine'
     )
+    twister_group.addoption(
+        '--clear',
+        dest='clear',
+        action='store',
+        default='no',
+        choices=('no', 'delete', 'archive'),
+        help='Clear twister artifacts. '
```

Comment:
I wondering if using `artifacts`/`artifactory` term is good idea. This introduces new terminology, and for me it is not clear if this term concerns only building images or maybe each output files, like reports and other stuff. Maybe better idea is using consistently `output directory` instead of `artifacts`.

Reply:
It is called artifacts in V1: https://github.com/zephyrproject-rtos/zephyr/blob/da2305081242b61067784a9f0c034f0105bc31a8/scripts/twister#L258